### PR TITLE
Update to PSPDFKit 10 for iOS

### PR DESCRIPTION
**Pull request by akashivskyy**
This PR updates the wrapper for PSPDFKit 10, Titanium SDK 9.1.0.GA and fixes issues when building with Xcode 12.

### CODE REVIEW

**Comment by akashivskyy** [comment]
The CI job will fail until PSPDFKit 10 for iOS is released.

**Review by radazzouz — APPROVED** [review]
LGTM 👍 I tested locally with the nightly build and it works great!

File: README.md [hunk 1]
```diff
@@ -218,6 +218,29 @@ $ npx appc --help
...
+```
+
+If you see one of the above errors while building or using PSPDFKit for iOS Titanium module, this is because Titanium SDK doesn't support building for Apple Silicon yet. To resolve these errors, make sure that you're using the latest version of the module.
+
+If the issue persists, try excluding the `arm64` architecture slice from your iOS Simulator builds until Titanium SDK officially supports Apple Silicon. You can do it by editing your project's build settings in Xcode or in an `xcconfig` file:
+
+```bash
+# Exclude the Apple Silicon architecture when building for iOS Simulator.
+EXCLUDED_ARCHS[sdk=iphonesimulator*] = $(inherited) arm64
+```
```
Comment (radazzouz) [inline README.md:242]: ❤️

**Comment by akashivskyy** [comment]
Rerunning the CI job and merging when green. 🙂

**Comment by radazzouz** [comment]
It looks like CI failed because of CocoaPods. There are known issue with CocoaPods 1.9.3. I would update CocoaPods to 1.10.0.rc.1

**Comment by akashivskyy** [comment]
That's what's on GitHub machines. I used a flag in Podfile that was introduced in 1.10, so I'll have to lock its version anyway.

**Comment by akashivskyy** [comment]
For the record: I decided not to lock CocoaPods versions while it's still pre-release. We only use CocoaPods to resolve and download the binaries, integration is turned off. So we can use whatever latest stable version there is, the issues with 1.9.3 you mentioned should not be affecting the setup in this case.

**Pull request merged**